var/spack/repos/builtin/packages/openturns/package.py

```diff
@@ -20,6 +20,7 @@ class Openturns(CMakePackage):
     git = "https://github.com/openturns/openturns.git"
     maintainers("liuyangzhuan")
 
+    version("1.20", sha256="2be5247f0266d153619b35dfb1eeeb46736c502dad993b40aff8857d6314f293")
     version("1.18", sha256="1840d3fd8b38fd5967b1fa04e49d8f760c2c497400430e97623595ca48754ae0")
     version("master", branch="master")
```

Comment:
```suggestion
    version("master", branch="master")
    version("1.20", sha256="2be5247f0266d153619b35dfb1eeeb46736c502dad993b40aff8857d6314f293")
    version("1.18", sha256="1840d3fd8b38fd5967b1fa04e49d8f760c2c497400430e97623595ca48754ae0")
```

@mschouler not something you touched in your PR but just standardizing the package, I noticed that the master version was hanging out at  the bottom of the versions instead of at the top.